Sources/Swift/Tools/LoadValidator.swift

```diff
@@ -0,0 +1,78 @@
+@_implementationOnly import _SentryPrivate
+import Darwin
+import Foundation
+import MachO
+
+@objc
+@_spi(Private) public final class LoadValidator: NSObject {
+    // Any class should be fine, ObjC classes are better
+    static let targetClassName = NSStringFromClass(SentryDependencyContainerSwiftHelper.self)
+    
+    // This function is used to check for duplicated SDKs in the binary.
+    // Since `SentryBinaryImageInfo` is not public and only available through the Hybrid SDK, we use the expanded parameters.
+    @objc
+    @_spi(Private) public class func checkForDuplicatedSDK(imageName: String, imageAddress: NSNumber, imageSize: NSNumber, dispatchQueueWrapper: SentryDispatchQueueWrapper) {
+        internalCheckForDuplicatedSDK(imageName, imageAddress.uint64Value, imageSize.uint64Value,
+                                      objcRuntimeWrapper: SentryDependencyContainerSwiftHelper.objcRuntimeWrapper(),
```

Comment:
`m`: For consistency I think we should also remove the default parameter for the `objcRuntimeWrapper` and pass it in via the method.

Reply:
🤦 I think we already had that discussion